README.md

```diff
@@ -113,115 +122,92 @@ We can delete a node e.g. the Medium reviewer's rankings
    documentClient.delete(tommyLeeJonesDBKey,"$.best_films_ranked[1]");
 ```
 
-### Document API
+### JsonPath Queries
 
-Below is the interface against which the API has been written
+JsonPath queries is java DSL for reading JSON documents.
```

Comment:
[JsonPath](https://goessner.net/articles/JsonPath/) is a query language for JSON.

Reply:
I took it directly from the official JsonPath repo:
https://github.com/json-path/JsonPath

Reply:
I think we can omit the details of the underlying Java library that we are using to fill in the functionality gaps, but rather introduce the user to the JsonPath specification.